html/static_index.html.haml

```diff
@@ -0,0 +1,76 @@
+!!!
```

Comment:
Yes, I thought so too, but couldn't come up with a good switch to toggle the two blocks which are missing from the static html template.

Additionally static export is using an instance variable and the server is using `description`, but that might be fixed by an `attr_reader`.